plothist/comparison.py

```diff
@@ -301,25 +296,19 @@ def get_ratio(
     _check_uncertainty_type(h1_uncertainty_type)
     _check_binning_consistency([h1, h2])
 
-    comparison_values = np.where(
-        h2.values() != 0, h1.values() / h2.values(), np.nan
-    )
+    comparison_values = np.where(h2.values() != 0, h1.values() / h2.values(), np.nan)
 
     if h1_uncertainty_type == "asymmetrical":
         uncertainties_low, uncertainties_high = get_asymmetrical_uncertainties(h1)
 
     if ratio_uncertainty_type == "uncorrelated":
         if h1_uncertainty_type == "asymmetrical":
             h1_high = h1.copy()
-            h1_high[:] = np.c_[h1_high.values(), uncertainties_high ** 2]
+            h1_high[:] = np.c_[h1_high.values(), uncertainties_high**2]
             h1_low = h1.copy()
-            h1_low[:] = np.c_[h1_low.values(), uncertainties_low ** 2]
-            comparison_uncertainties_low = np.sqrt(
-                get_ratio_variances(h1_low, h2)
-            )
-            comparison_uncertainties_high = np.sqrt(
-                get_ratio_variances(h1_high, h2)
-            )
+            h1_low[:] = np.c_[h1_low.values(), uncertainties_low**2]
+            comparison_uncertainties_low = np.sqrt(get_ratio_variances(h1_low, h2))
+            comparison_uncertainties_high = np.sqrt(get_ratio_variances(h1_high, h2))
```

Comment:
Is this fixed now? every time when one of us applied black this modification happened.